dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/singleevent/SingleEventOperationParams.java

```diff
@@ -91,6 +91,13 @@ public class SingleEventOperationParams {
 
   private boolean includeRelationships;
 
+  private boolean synchronizationQuery;
+
+  private Map<String, Set<String>> skipSyncDataElementsByProgramStage = new HashMap<>();
+
+  /** Indicates a point in the time used to decide the data that should not be synchronized */
+  private Date skipChangedBefore;
```

Comment:
Can we create a private setter to hide the lombok one so this can only be set using `builderForDataSync`?